pkg/services/folder/folderimpl/folder.go

```diff
@@ -992,6 +992,12 @@ func (s *Service) DeleteLegacy(ctx context.Context, cmd *folder.DeleteFolderComm
 			}
 		}
 
+		err = s.store.Delete(ctx, []string{cmd.UID}, cmd.OrgID)
+		if err != nil {
+			s.log.InfoContext(ctx, "failed deleting folder", "org_id", cmd.OrgID, "uid", cmd.UID, "err", err)
+			return err
+		}
+
```

Comment:
This change is a blocker for this PR, so I have it in the PR, but I'm having it reviewed here: https://github.com/grafana/grafana/pull/99399. Will have that merged in and merge in main here before merging this PR in